yarn.lock

```diff
@@ -153,6 +153,18 @@
   resolved "https://registry.yarnpkg.com/@polkadot/wasm-crypto/-/wasm-crypto-1.2.1.tgz#2189702447acd28d763886359576c87562241767"
   integrity sha512-nckIoZBV4nBZdeKwFwH5t7skS7L7GO5EFUl5B1F6uCjUfdNpDz3DtqbYQHcLdCZNmG4TDLg6w/1J+rkl2SiUZw==
 
+"@sindresorhus/is@^0.14.0":
```

Comment:
which package did you add that modifies yarn.lock? I see no change in package.json.

Reply:
The two things I did that might have affected this was that I had `nodemon` for a little bit as a dependency, but later removed from the package.json. I also ran `yarn upgrade`. But I am not exactly sure which would cause yarn.lock to be modified here.